config.json

```diff
@@ -266,7 +266,7 @@
         "name": "Linked List",
         "uuid": "ec60a578-8889-46a1-b7b8-306dbd8551d5",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["classes", "while-loops"],
```

Comment:
I don't know if we'll have while-loops as separate concept. Maybe just loops?

Reply:
In the big list we have while-loops plus we decided to go for for-loops, so I'd insist on while-loops